lms/djangoapps/courseware/date_summary.py

```diff
@@ -182,7 +182,11 @@ def is_enabled(self):
     @property
     def description(self):
         if datetime.now(pytz.UTC) <= self.date:
-            return _('To earn a certificate, you must complete all requirements before this date.')
+            (mode, is_active) = CourseEnrollment.enrollment_mode_for_user(self.user, self.course.id)
```

Comment:
Nitpick: you can drop these parentheses; they aren't necessary during assignment.